.github/workflows/release.yml

```diff
@@ -5,7 +5,7 @@
 on:
   push:
     branches:
-      - main
+      - beta/new_java_build
```

Comment:
This is right, but you need to change the target branch of this PR to match this one :)

Reply:
there is already a a release open, for this branch, i think it works as intended, i will just not merge this task, i will just update the branch, and have a snapshot release, without a merge.

Reply:
Ah I didn't see this matched your source branch too :rofl: